test/integration/taskRequests.test.js

```diff
@@ -639,6 +639,369 @@ describe("Task Requests", function () {
     });
   });
 
+  describe("PATCH /taskRequests/ - updates task request", function () {
+    let activeUserId, oooUserId;
+
+    describe("When the user is super user", function () {
+      beforeEach(async function () {
+        userId = await addUser(member);
+        activeUserId = await addUser(activeMember);
+        oooUserId = await addUser(member2);
+        superUserId = await addUser(superUser);
+
+        jwt = authService.generateAuthToken({ userId: superUserId });
+        sinon.stub(authService, "verifyAuthToken").callsFake(() => ({ userId: superUserId }));
+
+        taskId = (await tasksModel.updateTask(taskData[4])).taskId;
+
+        await userStatusModel.updateUserStatus(userId, idleUserStatus);
+        await userStatusModel.updateUserStatus(activeUserId, activeUserStatus);
+        await userStatusModel.updateUserStatus(oooUserId, { ...oooUserStatus });
+        await taskRequestsModel.addOrUpdate(taskId, userId);
+      });
+      afterEach(async function () {
+        sinon.restore();
+        await cleanDb();
+      });
+
+      it("should match response for successful approval", function (done) {
+        sinon.stub(taskRequestsModel, "approveTaskRequest").resolves({ approvedTo: member.username, taskRequest: {} });
+        chai
```

Comment:
Why is there a need to stub models, we should avoid stubs, mocks
We are have a in-memory Db for testing, we should be using that